app/megatron/connections/slack.py

```diff
@@ -371,12 +371,13 @@ def add_forward_footer(self, msg, user_data):
 
     def _refresh_access_token(self, platform_user_id):
         megatron_user = MegatronUser.objects.first()
-        response = safe_requests.post(
+        response = requests.post(
```

Comment:
Why'd you go back to `requests` from `safe_requests`?

Reply:
@jroblak I was getting a DNS lookup error but I found the solution, will push it now